### PR TITLE
feat: component/modal

### PR DESCRIPTION
**Pull request by asdfsena**
I've created basic modal component and will be create another variant for the modal soon!
You can see the preview [here](https://www.loom.com/share/6c70d53a195d4ac6bb1ac4ab3e0fa835?sid=889e1fd0-a82a-4447-aaab-26aafad28542)

### CODE REVIEW

File: src/components/Chip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      </span>
+      {endIcon ? endIcon : null}
+    </div>
+  );
+}
+
+const ShineAnimation = ({
+  color,
+  isClicked,
+}: {
```
Comment (rashgaroth) [inline src/components/Chip/index.tsx]: please put the type into the separated type (`export type ShineAnimation`) on the `components.ts`

File: src/utils/chip.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+export function getChipColor(
```
Comment (rashgaroth) [inline src/utils/chip.ts]: i think you better move this function into useChipController (create a new hooks)

File: src/utils/chip.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  color?: 'info' | 'success' | 'danger' | 'warning' | string,
+  outlined?: boolean,
+) {
+  console.log(color, outlined, 'getcolor');
```
Comment (rashgaroth) [inline src/utils/chip.ts]: please remove the console

**Review by rashgaroth — CHANGES_REQUESTED** [review]

File: src/components/Chip/index.tsx (hunk 1 above)
Comment (ayasena) [reply src/components/Chip/index.tsx]: alright, sorry forgot that

**Pull request merged**